source/install.sql

```diff
@@ -133,10 +137,12 @@ whenever oserror exit failure rollback
 
 
 prompt Validating installation
+-- erors only. ignore warnings
 select * from user_errors 
 where name not like 'BIN$%'  --not recycled
 and (name like 'UT%' or name in ('BE_FALSE','BE_LIKE','BE_NOT_NULL','BE_NULL','BE_TRUE','EQUAL','MATCH')) -- utplsql objects
-and attribute = 'ERROR'; -- erors only. ignore warnings
+and attribute = 'ERROR'
+/
```

Comment:
Is this change making the SQL display what is failing?
I've noticed that the current install script doesn't show errors, even if they exist.